queries.sql

```diff
@@ -46,3 +46,74 @@ select full_name, count(name) from owners left join animals on animals.owner_id=
 select animals.name from owners join animals on animals.owner_id= owners.id where owners.full_name='Jennifer Orwell' and animals.species_id=2;
 select animals.name from owners join animals on animals.owner_id= owners.id where owners.full_name='Dean Winchester' and animals.escape_attempts=0;
 select species.name, count(animals.name) from species join animals on animals.species_id= species.id group by species.name;
+
+
```

Comment:
- [ ] It would be better if the screenshot is visible from the point of view of the queries. It would be better not to include the screenshot files or the code. We need to see only the result of the queries. Also, to make it more readable, kindly rename the files to the specific queries to easily identify them. For example; the query `SELECT * FROM VETS` as shown in the screenshot is not part of the queries that we are expecting